routing_model/index.html

```diff
@@ -516,7 +517,7 @@ <h2>Resource proof from a destination's point of view</h2>
 
 
       DiscardRPC[Discard RPC]
-      RPC -- ResourceProofResponse --> DiscardRPC
+      RPC -- ResourceProofResponse<br/>from CANDIDATE<br/><br/>VOTED_ONLINE==yes --> DiscardRPC
```

Comment:
It is more a ResourceProofResponse as we want to discard the one we get from not candidate.